tests/unit/metadata/test_table.py

```diff
@@ -12,6 +13,168 @@
+        - Category tuple of category name and parameters passed to the method creating fake values.
+        Output:
+        - Fake values created with the specified method from the Faker object.
+        Utilizing the arguments given to it.
+        """
+        # Setup
+        metadata_dict = {
+            'fields': {
+                'foo': {
+                    'type': 'categorical',
+                    'pii': True,
+                    'pii_category': 'ean'
+                }
+            }
+        }
+        metadata = Table.from_dict(metadata_dict)
+
+        # Run
+        fake_8_ean = metadata._get_faker_fn(faker.Faker(), ("ean", 8))
+        ean_8 = fake_8_ean()
+
+        fake_13_ean = metadata._get_faker_fn(faker.Faker(), ("ean", 13))
+        ean_13 = fake_13_ean()
+
+        # Assert
+        assert len(ean_8) == 8
+        assert len(ean_13) == 13
+
+    def test__make_anonymization_mappings_unique_faked_value_in_field(self):
+        """Test that ``_make_anonymization_mappings`` method creates mappings for anonymized values.
+
+        The ``_make_anonymization_mappings`` method should map equal values in the original data
+        to the same faked value.
+
+        Input:
+        - DataFrame with a field that should be anonymized based on the metadata description.
+        Output:
+        - The mappings created from the original values to faked values.
+        """
+        # Setup
+        metadata_dict = {
+            'fields': {
+                'foo': {
+                    'type': 'categorical',
+                    'pii': True,
+                    'pii_category': 'email'
+                }
+            }
+        }
+        data = pd.DataFrame({
+            'foo': ['test1@example.com', 'test2@example.com', 'test1@example.com']
+        })
+        metadata = Table.from_dict(metadata_dict)
+
+        # Run
+        foo_mappings = metadata._make_anonymization_mappings(data)["foo"]
+
+        # Assert
+        assert len(foo_mappings) == 2
+        assert list(foo_mappings.keys()) == ['test1@example.com', 'test2@example.com']
+
+    @pytest.mark.skipif(faker.VERSION < "3.0.0", reason="Higher version of Faker required.")
+    def test__make_anonymization_mappings_multiple_localizations(self):
+        """Test that the ``_make_anonymization_mappings`` method takes localizations into account
+        when creating mappings for anonymized values.
+
+        The ``_make_anonymization_mappings`` method should create localized fake values if
+        `pii_locales` is specified in the metadata description.
+
+        Input:
+        - DataFrame with a field that should be anonymized based on the metadata description.
+        Metadata description specifies multiple localizations using `pii_locales`.
+        Output:
+        - The mappings created from the original values to localized faked values.
+        """
+        # Setup
+        def _mock_faker_getattr(obj, fn_name):
```

Comment:
Is it also possible to define the mock function outside of the tests functions?
The same mock is used multiple times and for me it would make sense to pull this outside of the Test setup of each test.

Like this:
```
    def _mock_faker_getattr(obj, fn_name):
        if fn_name == "company":
            return lambda: obj.__lang__
        else:
            return getattr(obj, fn_name)

    @patch('faker.generator.getattr', _mock_faker_getattr)
```

Then no mock must be created / changed inside the test.